workspaces/cms-scripts/src/index.ts

```diff
@@ -45,32 +45,43 @@ for (const simpleData of simpleDataTypes) {
 }
 
 const simpleFiles = [
-  await getSimpleFiles("dapps"),
-  await getSimpleFiles("wallets"),
-  await getSimpleFiles("block-explorers"),
-  await getSimpleFiles("bridges"),
-  await getSimpleFiles("fiat-on-ramps"),
-  await getSimpleFiles("redirects"),
-  await getSimpleFiles("alert"),
+  await getSimpleFiles("settings", "dapps"),
+  await getSimpleFiles("settings", "wallets"),
+  await getSimpleFiles("settings", "block-explorers"),
+  await getSimpleFiles("settings", "bridges"),
+  await getSimpleFiles("settings", "fiat-on-ramps"),
+  await getSimpleFiles("settings", "redirects"),
+  await getSimpleFiles("settings", "alert"),
+  await getSimpleFiles("seo", "events", true),
+  await getSimpleFiles("seo", "tutorials", true),
+  await getSimpleFiles("seo", "jobs", true),
+  await getSimpleFiles("seo", "footer", true),
+  await getSimpleFiles("seo", "home", true),
+  await getSimpleFiles("seo", "language", true),
+  await getSimpleFiles("seo", "search", true),
 ];
 
 for (const simpleFile of simpleFiles) {
+  const resourceDir =
+    simpleFile.collectionName === "settings"
+      ? simpleFile.resourceName
+      : `${simpleFile.collectionName}/${simpleFile.resourceName}`;
+
   try {
-    await fs.mkdir(`_crowdin/data/${simpleFile.resourceName}`, {
+    await fs.mkdir(`_crowdin/data/${resourceDir}`, {
       recursive: true,
     });
 
     for (const locale of locales) {
       const data = simpleFile.localeMap.get(locale)!;
 
       await write(
-        `_crowdin/data/${simpleFile.resourceName}/${locale}.json`,
-        data.items,
+        `_crowdin/data/${resourceDir}/${locale}.json`,
+        simpleFile.writeRootData ? data : data.items
       );
     }
   } catch (err) {
```

Comment:
let's remove try catch (script should just fail if there is an error)

Reply:
@thecotne Done!